setup.py

```diff
@@ -37,13 +37,14 @@
       python_requires='>=3.6',
       test_suite="tests",
       install_requires=[
+          "idna==2.10",  # Required to thread version needle with requests library
```

Comment:
Is this going to come back and bite us in the butt at some later time?

Reply:
Added safety check to make sure the bite isn't fatal